tools/packaging/common/envoy_bootstrap.json

```diff
@@ -256,14 +291,8 @@
               ],
               "validation_context": {
                 "trusted_ca": {
-                  {{- if .provisioned_cert }}
-                  "filename": "{{(printf "%s%s" .provisioned_cert "/root-cert.pem") }}"
-                  {{- else if eq .pilot_cert_provider "kubernetes" }}
-                  "filename": "./var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-                  {{- else if eq .pilot_cert_provider "istiod" }}
-                  "filename": "./var/run/secrets/istio/root-cert.pem"
-                  {{- end }}
-                },
+                  "filename": "{{- .provisioned_cert }}"
```

Comment:
provisioned_cert feels like the wrong name now.. seems like it should be xds_root_cert or similar

Reply:
Agreed. Can we bike shed the new name in separate PR ?